#################################
# PID parameters for pose seeking
#################################

# Maximum value of error integral for all PID controllers
windup_limit: 0.05

# PID gains
x_proportional_gain: 5
y_proportional_gain: 5
z_proportional_gain: 5
x_integral_gain: 0.0
y_integral_gain: 0.0
z_integral_gain: 0.0
x_derivative_gain: 0.2
y_derivative_gain: 0.2
z_derivative_gain: 0.2

angular_proportional_gain: 10.0
angular_integral_gain: 0.8
angular_derivative_gain: 0.0